node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log(process.argv.slice(2).join(''));
```

Comment:
Nice! A bit overkill for the requirements but a clever way to display all of the user arguments!